tests/foreman/ui_airgun/test_contentview.py

```diff
@@ -2058,3 +2059,92 @@ def test_positive_composite_child_inc_update(session):
             assert len(packages) > 1
             packages_data = set('{}-{}-{}.{}.rpm'.format(*row.values()) for row in packages)
             assert FAKE_0_INC_UPD_NEW_PACKAGE in packages_data
+
+
+@tier3
+def test_positive_module_stream_end_to_end(session, module_org):
+    """Create content view with custom module_stream contents, publish and promote it
+    to Library +1 env. Then disassociate repository from that content view
+
+    :id: 66955a89-14ed-414e-a15a-6ed9ede520ea
+
+    :steps:
+        1. Create yum repo with module_stream content and sync it
+        2. Create content view and add created repo to it
+        3. Publish that content view
+        4. Promote it to next environment
+
+    :expectedresults: Content view works properly with module_streams and
+        count shown should be correct
+
+    :CaseLevel: System
+    """
+    repo_name = gen_string('alpha')
+    env_name = gen_string('alpha')
+    cv_name = gen_string('alpha')
+    # Creates a CV along with product and sync'ed repository
+    create_sync_custom_repo(module_org.id, repo_name=repo_name,
```

Comment:
```
create_sync_custom_repo(
    module_org.id, 
    repo_name=repo_name,
    repo_url=CUSTOM_MODULE_STREAM_REPO_2)
```

Reply:
Second: 
Is module stream repo has `REPO_TYPE['yum']` type? Or it has its own unique type?

Reply:
REPO_TYPE['yum'] is correct, module streams derived from yum repos only